vcd/resource_vcd_network_routed.go

```diff
@@ -253,10 +253,10 @@ func resourceVcdNetworkRead(d *schema.ResourceData, meta interface{}) error {
 	if c := network.OrgVDCNetwork.Configuration; c != nil {
 		d.Set("fence_mode", c.FenceMode)
 		if c.IPScopes != nil {
-			d.Set("gateway", c.IPScopes.IPScope.Gateway)
-			d.Set("netmask", c.IPScopes.IPScope.Netmask)
-			d.Set("dns1", c.IPScopes.IPScope.DNS1)
-			d.Set("dns2", c.IPScopes.IPScope.DNS2)
+			d.Set("gateway", c.IPScopes.IPScope[0].Gateway)
+			d.Set("netmask", c.IPScopes.IPScope[0].Netmask)
+			d.Set("dns1", c.IPScopes.IPScope[0].DNS1)
+			d.Set("dns2", c.IPScopes.IPScope[0].DNS2)
```

Comment:
Why have we made this an array, but using only the first occurrence?

Reply:
The API requires an array in the configuration structure.